modern/src/App.js

```diff
@@ -33,8 +33,15 @@ function handleMouseEventDispatch(node, event, eventName) {
   // In order to properly calculate the x/y coordinates like MAKI does we need
   // to find the container element and calculate based off of that
   const container = Utils.findParentNodeOfType(node, ["container"]);
-  const x = event.clientX - container.getleft();
-  const y = event.clientY - container.gettop();
+  let x, y;
+  if (container) {
```

Comment:
Maybe we should just add a utility function which traverses up but also considers the current item?

Reply:
Sounds good...Any ideas for...naming? `findParentOrCurrentNodeOfType` :-X

Reply:
Yeah, I cans think of a good name either. That one’s fine for now.